### PR TITLE
Added docs

### LINKED ISSUE #120: Document the internal structure of a PoN
- [ ] What are the fields?  Particularly, what is the log-normals field?
- [ ] What does each field mean?  
- [ ] Example in matlab of loading a PoN file

### PR DESCRIPTION
**Pull request by LeeTL1220**
Matlab script example is in another branch (PoN QC)

closes #120

### CODE REVIEW

**Comment by LeeTL1220** [comment]
@davidbenjamin Can you please ignore the failing unit test (for obvious reasons)?

File: docs/CNVs/CNVPoNFields.v5.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- ``log_normals_pinv`` -- pseudoinverse of ``log_normals``.  Size is transpose of ``log_normals``.
+- ``log_normals_targets`` -- the targets used in ``log_normals`` as a string array, in row major form.  See ``num_target_cols``
+- ``num_target_cols`` -- targets are encoded as an array, even though a table makes more sense.  This is the number of columns in any target field.  Stored as a single number (3.0) Size: [1x1]
+- ``raw_targets`` -- Unfiltered target list that was used as input to the CreatePanelOfNormals call.  These are stored as string array, in row major form.  See ``num_target_cols``.  Size [initial_targets x 3 (num_target_cols)]
+- ``version`` -- PoN version.  Stored as a single double (5.0) Size:  [1 x 1]
+- ``reduced_pon`` -- SVD of ``log_normals`` with high variance components (columns) preserved.  The number of preserved columns (eigen_samples) depends on the mean of the eigenvalues.  Size:  [targets x eigen_samples]  The corresponding targets are in ``log_normals_targets``.
```
Comment (davidbenjamin) [inline docs/CNVs/CNVPoNFields.v5.0.md]: This makes it sound like reduced_pon *is* the SVD when in fact it is the eigensamples that we retain based on the SVD.

Comment (davidbenjamin) [inline docs/CNVs/CNVPoNFields.v5.0.md]: Personally, I prefer "eigensamples" without a space for consistency with math, but I won't force my opinion unless others find it compelling.

**Comment by davidbenjamin** [comment]
@LeeTL1220 I found one line slightly confusing.  Rewrite as you see fit and merge.

**Pull request merged**